opentech/apply/funds/admin.py

```diff
@@ -75,6 +97,7 @@ class ApplyAdminGroup(ModelAdminGroup):
     menu_icon = 'folder-open-inverse'
     items = (
         RoundAdmin,
+        ScreeningStatusAdmin,
```

Comment:
I think putting it at the top or bottom of the list is probably more reasonable.